data/core/faqs.md

```diff
@@ -530,6 +530,29 @@
     - action_faqs
     - utter_direct_to_step2
 
+## new to rasa + not new to chatbots + not migrating
```

Comment:
i don't think this story needs to be here -- the FAQ part won't impact the later stories